### PR TITLE
bokeh 2.4.2 for py10 all platforms

### PR DESCRIPTION
**Pull request by cbouss**
dask requires bokeh 2.4.2.
bokeh 2.4.2 is missing py10 packages for all architectures

### CODE REVIEW

File: abs.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+channels:
+  - c3i_test2
```
Comment (katietz) [inline abs.yaml:2]: is there a specific reason why we add here this channel?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -10,6 +10,7 @@ source:
 
 build:
   number: 0
+  # trigger: 1
```
Comment (katietz) [inline recipe/meta.yaml:13]: another issue here, while you are on it.  adjust test invocation of pip. then you can raise builder number too.  As you modified recipe, there will be an new hash valure anyway

**Review by katietz — CHANGES_REQUESTED** [review]

File: abs.yaml (hunk 1 above)
Comment (cbouss) [reply abs.yaml:2]: This way bokeh is uploaded to c3i_test2 by prefect. Which is a kind of staging and will allow publishing dask and all its dependencies at once.
But I will have to remove this once all is uploaded.

File: recipe/meta.yaml (hunk 2 above)
Comment (cbouss) [reply recipe/meta.yaml:13]: What is the recommended invocation?
Should it be this one?
{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv

Comment (cbouss) [reply recipe/meta.yaml:13]: ok, that was for pip check

**Review by varlackc — COMMENTED** [review]
We probably should use the `https` instead of `http` for the `home url`, `doc_url` and `dev_url`.
https://github.com/AnacondaRecipes/bokeh-feedstock/blob/cb_bokeh_2.4.2_py10/recipe/meta.yaml#L65&&L79

**Comment by cbouss** [comment]
> We probably should use the `https` instead of `http` for the `home url`, `doc_url` and `dev_url`. https://github.com/AnacondaRecipes/bokeh-feedstock/blob/cb_bokeh_2.4.2_py10/recipe/meta.yaml#L65&&L79

Thanks, I updated the whole meta section.

**Review by tobijk — APPROVED** [review]

**Pull request merged**